IBM/lale
# Experimental task graph improvements.

- handle non-incremental estimators
- test cross_val_score
- clean up arguments to cross_val_score
- change visualization node style for train tasks
- delete the output of tasks as soon as they are no longer needed
